<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Binary Translator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{

background-color: rgb(74, 74, 74)

}</string>
  </property>
  <widget class="QLabel" name="IntroMSGLabel">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>50</y>
     <width>350</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono ExtraBold</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{

color: 'lime'


}</string>
   </property>
   <property name="text">
    <string>Please select which translation you want</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="TextToBinaryButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>300</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono NL ExtraBold</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(54, 54, 54);
border-radius: 20px;
color: 'lime';
border-style: outset;</string>
   </property>
   <property name="text">
    <string>Text to Binary</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BinaryToTextButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>390</y>
     <width>300</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono NL ExtraBold</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(54, 54, 54);
border-radius: 20px;
color: 'lime';
border-style: outset;</string>
   </property>
   <property name="text">
    <string>Binary to text</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
